modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
+
+Operators can manage other Operators within their component. This lends to a
+transparent model: a single Operator usually manages a single binary file.
+Operators also offer a more granual configuration experience. You configure each component by
+modifying its Operator instead of modifying a global configuration file.
+
+In version 4.0, separate processes and images and pods run to drive the kubernetes
+API and pod managers. The control plane services run as static pods so they can
+manage normal workloads or processes the same way that they manage disaster
+recovery. Aside from the core control plane components, other services run as 
+normal pods on the cluster, managed by regular Kubernetes constructs. Unlike in the past
+where the `kubelet` could be running as containerized or non-containerized, the `kubelet`
+always runs as a `systemd` process.
+
+
+[id='cluster-version-operator-{context}']
```

Comment:
I don’t want to focus on these lower level details until we have an architecture topic on “upgrades” (which covers most of the details).  I would do link outs to the major functional areas and list a few of the core operators and their roles, but this is too focused on “what” and not “why”.  The details below would be maybe a subtopic under specific operators but as is this is too messy.